packages/cloud-cognitive/src/components/CreateModal/CreateModal.test.js

```diff
@@ -141,8 +138,60 @@ describe(componentName, () => {
   });
 
   it('applies focus to selected element', () => {
-    const { container } = renderComponent({ primaryFocus });
-    const firstInput = container.querySelector(primaryFocus);
+    const { container } = renderComponent(
+      { selectorPrimaryFocus },
+      <TextInput
+        key="form-field-1"
+        id="1"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />
+    );
+    const firstInput = container.querySelector(selectorPrimaryFocus);
     expect(firstInput === document.activeElement).toBeTruthy();
   });
+
+  it('throws an error if there are more than 4 child nodes inside of the modal', () => {
+    const { container } = renderComponent({}, [
```

Comment:
This prevents the propTypes type error coming out on the console during the test run.

```suggestion
    const error = jest.spyOn(console, 'error').mockImplementation(() => {});

    const { container } = renderComponent({}, [
```